content/articles/concept-of-first-hop-redundancy-protocol/index.md

```diff
@@ -0,0 +1,133 @@
+author: rabo-james-bature
+date: 2021-06-10T00:00:00-18:00
+topics: [Networking]
+excerpt_separator: <!--more-->
+images:
+
+   url: hero.jpg
+   alt: First Hop Redundancy Protocol example image
+
+Organizations use various protocols to prevent a single point failure in their works. One of these protocols is the First Hop redundancy Protocol (FHRP) which presents a virtual default gateway to the organization's network to provide a near 100 percent network uptime.
+
+This article will look at how FHRP is used to achieve redundancy in networks, variety of FHRP, and then configure Hot Standby Redundancy Protocol (HSRP), a Cisco proprietary protocol used in the implementation of FHRP.
+
+### Table of contents
+
+- [Goals](#goals)
+- [Prerequisite](#prerequisite)
+- [Requirements](#requirements)
+- [Default gateway](#default-gateway)
+- [First Hop Redundancy Protocol (FHRP)](#first-hop-redundancy-protocol-fhrp)
+- [Various First Hop Redundancy Protocols](#various-first-hop-redundancy-protocols)
+  - [Host Standby Router Protocol (HSRP)](#host-standby-router-protocol-hsrp)
+  - [Gateway Load Balancing Protocol (GLBP)](#gateway-load-balancing-protocol-glbp)
+  - [ICMP Router Discovery Protocol (IRDP)](#icmp-router-discovery-protocol-irdp)
+- [HSRP features](#hsrp-features)
+  - [HSRP preemption](#hsrp-preemption)
+- [Steps to configuring HSRP](#steps-to-configuring-hsrp)
+- [Configuring HSRP protocol on a Cisco router](#configuring-hsrp-protocol-on-a-cisco-router)
+- [Conclusion](#conclusion)
+- [Activity File](#activity-file)
+- [Further reading](#further-reading)
+
+### Goals
+At the end of this article, the reader is expected to learn:
+- What FHRP is
+- Various FHRP options available
+- How to configure HSRP on a Cisco router.
+  
+### Prerequisite
+To better understand this article, the reader is expected to have a basic understanding of the Cisco Command Line Interface (CLI)
+
+### Requirements
+We will use the Cisco packet tracer software for this tutorial which one can download at [Download Packet Tracer](https://www.netacad.com/portal/resources/packet-tracer)
+
+### Default gateway
+In a network environment, devices must communicate with each other within the network and outside the network. A device that communicates with over devices outside its network or subnet requires a default gateway.
+A default gateway is a network node that allows seamless communication between networks, allowing one device to communicate with another on a different network or subnet. It is the first route taken by an Internet Protocol (IP) packets to ensure that internet requests are sent to the right destination.
+The node serving as a default gateway is usually is configured with an Internet Protocol (IP) address. Each client in the network or subnet is configured with the same IP address. Because of this, if the router or the layer three switch serving as a default gateway fails, all devices connected to it will be isolated from the outside network.
```

Comment:
Grammatical error. 

The node serving as a default gateway is configured with an Internet Protocol address.